bundle/edu.gemini.spdb.reports.collection/src/main/java/edu/gemini/dbTools/odbState/ObservationState.java

```diff
@@ -178,7 +178,7 @@ private static String _getNight(final long time) {
         final StringBuilder buf = new StringBuilder();
         buf.append(startDay).append("/").append(endDay).append(" ");
 
-        final DateFormat f = new SimpleDateFormat("MMM yyyy");
+        final DateFormat f = new SimpleDateFormat("yyyy-MMM-dd");
```

Comment:
This formatted date doesn't seem to escape an XML file into which it is written.  I don't know if anything parses that `String` somewhere else.  I think maybe we could just leave this one.